### PR TITLE
chooseSpawnPos => whitelist spawn positions

### LINKED ISSUE #83: Add white list for fn_chooseSpawnPos.sqf
Add a white list for markers, black list already implented.

### PR DESCRIPTION
**Pull request by TheShinriel**
resolve #83 

Add a whitelist parameter in function chooseSpawnPos

NOT TESTED YET

### CODE REVIEW

File: gdc_lib_main/functions/util/fn_chooseSpawnPos.sqf [hunk 1]
```diff
@@ -30,52 +43,79 @@ if ((count _units) == 0) then {
...
 // si le joueur a le rang requis il peut déplacer le marqeur avec un clic gauche sur la carte
 if (rank player == _rank) then {
-	[_mk,_blist,_water] onMapSingleClick {
+	[_mk, _blist, _wlist, _water] onMapSingleClick {
+		params ["_mk", "_blist", "_wlist", "_water"];
+
 		_valid = true;
+		// Check if whilisted area
+		// If no whitelist marker, it is valide because all map is valid
+		if(!count _wlist == 0) then {
```
Comment (Tanin69) [inline gdc_lib_main/functions/util/fn_chooseSpawnPos.sqf:52]: wrong position for ! operator. Must be !( and not (!

**Review by Tanin69 — CHANGES_REQUESTED** [review]

File: gdc_lib_main/functions/util/fn_chooseSpawnPos.sqf (hunk 1 above)
Comment (TheShinriel) [reply gdc_lib_main/functions/util/fn_chooseSpawnPos.sqf]: Must be
> if(count _wlist == 0) then {

The comment is clear, "NO marker" so == 0

File: gdc_lib_main/functions/util/fn_chooseSpawnPos.sqf [hunk 2]
```diff
@@ -30,52 +43,79 @@ if ((count _units) == 0) then {
...
+			__valid = false;
```
[the remark this answers is not in the document]
Comment (TheShinriel) [reply gdc_lib_main/functions/util/fn_chooseSpawnPos.sqf]: That's an error, that could worked but everwhere else there is only one underscore

**Review by TheShinriel — COMMENTED** [review]
Fix the errors with the last commit

**Pull request merged**